auth/auth-domain/src/test/java/shop/woowasap/auth/domain/PasswordTest.java

```diff
@@ -0,0 +1,54 @@
+package shop.woowasap.auth.domain;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import shop.woowasap.auth.domain.exception.PasswordValidateException;
+
+@DisplayName("유저 비밀번호 테스트")
+class PasswordTest {
+
+    @DisplayName("유저 비밀번호 정상생성 테스트")
+    @Test
```

Comment:
저희 컨벤션이 길이가 작은걸 위에 위치시키기로 했던거 같아요

```suggestion
    @Test
    @DisplayName("유저 비밀번호 정상생성 테스트")
```

Reply:
vlauesource는 길이에의해 마지막으로 보냈습니다. [d79c38d](https://github.com/woowasap/ASAP/pull/43/commits/d79c38d2b90d9dfce789526b64df13302c578eca)